### ISSUE TITLE
Efi Partition too small for Kernel Update

### ISSUE THREAD

**Issue by dnan0s** [gardenlinux/gardenlinux#1893]
**What would you like to be added**:
At some point you changed something with the efi Partition. On my latest Garden Linux Machines it's only 65MB large. With the kernel right after installation it's already 34MB/65MB used. Installation of another kernel fails kind of silently (you have to look into the log):

```
...
Setting up linux-image-6.1.68-gardenlinux-amd64 (6.1.68-0gardenlinux2) ...
/etc/kernel/postinst.d/dracut:
dracut: Generating /boot/initrd.img-6.1.68-gardenlinux-amd64
/etc/kernel/postinst.d/zz-systemd-boot:
Installing kernel version 6.1.68-gardenlinux-amd64 in systemd-boot...
install: error writing '/efi/Default/6.1.68-gardenlinux-amd64/initrd': No space left on device
Error: could not copy '/tmp/kernel-install.staging.cbUUJP/initrd' to '/efi/Default/6.1.68-gardenlinux-amd64/initrd'.```
...
```

```
root@gardenlinux:/home/myuser# lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
sda      8:0    0   30G  0 disk 
├─sda1   8:1    0   65M  0 part /efi
└─sda2   8:2    0 29.9G  0 part /
```

So, apt does not throw an error. after the upgrade no failed upgrade is offered again. After a reboot the old kernel is still used, which makes Kernel Updates basically fail. is there a good reason for it?

tl/dr: 65MB for efi size is in my opinion too small for Kernel Updates, as this example shows. Can you please increase it again? 

(Also: Where in the build script can i define the size of the efi partition myself?)

**Comment by nkraetzschmar** [comment]
During image build the size of all partitions gets calculated automatically to be as small as possible by default. This size can however be adjusted by setting an explicit size parameter using an `fstab.mod` script. See [features/_secureboot/fstab.mod](https://github.com/gardenlinux/gardenlinux/blob/a764ccb0e3e51923537be8d302fdfeb6564bdbc7/features/_secureboot/fstab.mod) for an example `fstab.mod` script adjusting the size of the efi partition.

**Comment by fwilhe** [comment]
Hi @dnan0s 

Did @nkraetzschmar 's comment resolve the issue for you? If so please close it, otherwise please let us know what the issue is.

Thanks.

**Comment by dnan0s** [comment]
Sorry guys, i kind of got lost of this issue. Now we had to deal with it again, so i added a PR. Hope it's fine.

**Comment by Vincinator** [comment]
@dnan0s sorry, for the late reply! I will take a look at your PR next week. re-opening this issue

**Comment by Vincinator** [comment]
@dnan0s  see my comment in your PR. 

I don't see that a bigger efi partition should be the default of metal just yet. 
As stated in my comment in your PR, I suggest to introduce a new feature of type modifier.
